ncl/ncl_entries/trigonometric_functions.ipynb

```diff
@@ -76,7 +80,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 47,
```

Comment:
Do you know if we've thought about any ways to filter these types of metadata changes out?

It l looks like we might be able to use nbconvert ([see here](https://nbconvert.readthedocs.io/en/latest/api/preprocessors.html#metadata-and-header-control)).  

Other options might look like moving to MyST MD or using [jupytext](https://github.com/mwouts/jupytext).

I'll log this as an issue.